changelog.json

```
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "2.9.5-beta3": [
+      "[Added] Warn when hidden bidi characters are present in diffs - #13343",
```

I think this text is a bit better maybe?
```suggestion
      "[Improved] Warn users when files contain bidirectional Unicode text - #13343",
```